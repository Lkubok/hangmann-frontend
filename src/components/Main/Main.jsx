import React, { Component } from "react";
import "./Main.scss";

export class Main extends Component {
  render() {
    const { children } = this.props;
    return <div className="app-main">{children}</div>;
  }
}
export default Main;
