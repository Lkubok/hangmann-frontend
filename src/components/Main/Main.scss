@import "../../scss/mixins.scss";
@import "../../scss/variables.scss";

.app-main {
  @include basic-app-padding;
  @include flex-cl-row;
  @include basic-app-margin;
  background-color: $dark-color;
  color: $white-color;
  min-height: 200px;
  padding-bottom: 50px;
  @media all and (max-width: 992px) {
    padding-bottom: 5px;
    }
}
