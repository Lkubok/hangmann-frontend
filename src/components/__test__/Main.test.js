import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { Main } from "../Main/Main";
Enzyme.configure({ adapter: new Adapter() });

describe("Main component", () => {
  beforeEach(() => {});
  it("Should match snapshot", () => {
    const wrapper = shallow(<Main />);
    expect(wrapper).toMatchSnapshot();
  });
});
