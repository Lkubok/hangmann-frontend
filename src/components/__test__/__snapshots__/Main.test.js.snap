// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main component Should match snapshot 1`] = `
<div
  className="app-main"
/>
`;
